server/src/main/java/org/apache/iotdb/db/writelog/io/SingleFileLogReader.java

```diff
@@ -139,4 +143,12 @@ public void open(File logFile) throws FileNotFoundException {
   public boolean isFileCorrupted() {
     return fileCorrupted;
   }
+
+  private void truncateBrokenLogs() {
+    try (FileChannel channel = new FileOutputStream(filepath, true).getChannel()) {
```

Comment:
channel close,  but FileOutputStream does not close.

Reply:
Change code like this?
`
try (FileOutputStream outputStream = new FileOutputStream(filepath, true);
        FileChannel channel = outputStream.getChannel()) {
`

Reply:
yes